src/config-hardware/audio-apps.md

```diff
@@ -0,0 +1,39 @@
+# Audio Applications & Extensions
+
+These applications provide control and customization settings for audio devices beyond the `Settings` menu.
+
+---
+
+## EasyEffects
+
+[EasyEffects](https://github.com/wwmm/easyeffects) is a third-party application that can apply audio effects and filters to audio input and output streams. You can download and install EasyEffects from the Pop!\_Shop.
+
+![EasyEffects](/images/audio-mic/easyeffects.png)
+
+## Audio Output Switcher
+
+[Audio Output Switcher](https://extensions.gnome.org/extension/751/audio-output-switcher/) is a GNOME extension that adds selectable audio input/output devices to the System menu. See [this section](/customize-pop/gnome-tweaks-extensions/gnome-extensions.md) for more information about installing GNOME Extensions.
+
+![Audio Output Switcher](/images/audio-mic/output-switcher.png)
+
+## Alsamixer
+
+Alsamixer is a graphical command-line program that directly interfaces with the Advanced Linux Sound Architecture (ALSA) framework. Alsamixer provides more advanced audio configuration options, like selecting sound cards and viewing all system audio devices. Alsamixer comes preloaded in Pop!_OS and does not require installation.
+
+### Launching Alsamixer
+
+Open the Terminal and type `alsamixer` to launch the program.
+
+![Launch Alsamixer](/images/audio-mic/launch-alsamixer.png)
+
+### Alsamixer Controls
+
+Many Alsamixer functions are controlled using keyboard commands. Press `F1` to view all Alsamixer controls.
+
+![Alsamixer Keyboard Controls](/images/audio-mic/alsamixer-controls.png)
+
+### Selecting a Soundcard
+
+Press `F6` and use the arrow keys to highlight a sound card. Press `Enter` to select the sound card.
+
+![Select Sound Card](/images/audio-mic/select-sound-card.png)
```

Comment:
This didn't really explain how to change any settings with alsamixer. You may want to at least include using the arrow keys to navigate and the `m` key to mute/unmute inputs and outputs. One of the only times I've ever seen the use of `alsamixer` recommended is to unmute volume streams (since those being muted can cause a lack of audio even when it appears unmuted to PulseAudio-- not sure if that still  applies with PipeWire.)

Reply:
Let me know your thoughts on this description: 

"AlsaMixer communicates directly with system-level audio processes, which makes it useful for muting or unmuting audio streams at a low level. Use the `🠐` `🠒` keys to navigate between sources, and press `m` to mute or unmute a source."

Reply:
"system-level" doesn't really mean anything; PipeWire is also a "system" daemon compared with end-user applications. How about something like this?

> ALSA controls device volumes at a lower level than PipeWire. It's recommended to control the volume using PipeWire, but if a sound device becomes muted in ALSA, `alsamixer` can be used to unmute it. Use the `🠐` `🠒` keys to navigate between sources, and press `m` to mute or unmute a source.

In addition to trying to be more clear, I'm also not wanting to give the impression that users should jump to `alsamixer` for regular/non-troubleshooting usage.